include/mbedtls/config.h

```diff
@@ -634,6 +634,23 @@
  */
 //#define MBEDTLS_AES_ONLY_ENCRYPT
 
+/**
+ * \def MBEDTLS_AES_SCA_COUNTERMEASURES
```

Comment:
Major - if we upstream this we shouldn't have a switch for just AES SCA countermeasures, but instead a generalised switch for "additional SCA countermeasures".